src/scss/styles-4-releases.scss

```diff
@@ -142,6 +142,11 @@ $tableblue-even: #1F2F4E;
   color: rgba(255, 255, 255, .7);
 }
 
+.tick img {
```

Comment:
I don't see this tick on staging?  Did you forget to commit the image file?  Probably better just to use a font for the tick?

Reply:
It's not there atm because our latest release just got deleted and is being re built. It worked before.